README.md

```diff
@@ -382,12 +408,20 @@ theme={{
     dayTextAtIndex6: {
       color: 'blue'
     }
+  },
+  'stylesheet.day.basic': {
+    dayTextAtIndex0: {
+      color: 'red'
+    },
+    dayTextAtIndex6: {
+      color: 'blue'
+    }
   }
 }}
 ```
 
 <kbd>
-  <img height=50 src="https://github.com/wix-private/wix-react-native-calendar/blob/master/demo/day-header-style.png?raw=true">
+  <img height=50 src="https://github.com/wix-private/wix-react-native-calendar/blob/master/demo/day-header-and-days-style.png?raw=true">
```

Comment:
I have no ideas, why it automatically formatted the README file but the actual changes that I made is just here